### PR TITLE
fix(operators): added tips after operator description

### PR DESCRIPTION
**Pull request by btroncone**
Added tips to display after operator long description if they exist. Also removed parameter section
if no parameters exist.

### CODE REVIEW

File: src/app/operators/components/operator/operator.component.html [hunk 1]
```diff
@@ -21,12 +21,18 @@ <h3 class="short-description" [innerHTML]="shortDescription"></h3>
     [operatorExamples]="examples" class="margin-bottom-16">
   </app-operator-examples>
   <app-operator-parameters
-    [operatorParameters]="parameters">
+    [operatorParameters]="parameters"
+    *ngIf="paramaters?.length">
```
Comment (sumitarora) [inline src/app/operators/components/operator/operator.component.html]: Typo `parameters`

**Comment by ladyleet** [comment]
@sumitarora really appreciate you helping review PRs.

File: src/app/operators/components/operator/operator.component.html (hunk 1 above)
Comment (btroncone) [reply src/app/operators/components/operator/operator.component.html]: Oops! Thanks for catching, sending up a fix now!

**Comment by btroncone** [comment]
Weird, looks like everything passed this time but something with selenium failed. Care to take a look @kwonoj?

**Comment by kwonoj** [comment]
It seems one time woes on travis.

**Comment by btroncone** [comment]
Can this be reviewed / merged? It fixes a couple of display issues on the operator page that may cause confusion to others checking their operator PR's. @ladyleet @kwonoj  @sumitarora

**Comment by sumitarora** [comment]
@btroncone Please rebase with master.

**Comment by ladyleet** [comment]
@btroncone 👍  - waiting on @kwonoj or @ashwin-sureshkumar to review & merge.

**Pull request merged**